source/tutorial/routes-and-templates.md

```diff
@@ -77,7 +77,7 @@ Let's create another route with details for contacting the company.  Once again,
 ember g route contact
 ```
 
-We see that our generator has created a `contact` route in the `app/router.js` file, and a corresponding route handler in `app/routes/contact.js`.  We will be using the `contact` template, so the `contact` route does not need any additional changes.
+We see that our generator has created a `contact` route in the `app/router.js` file, and a corresponding route handler in `app/routes/contact.js`.  Since, we will be using the `contact` template, so the `contact` route does not need any additional changes.
```

Comment:
I think the _so_ after the comma should be removed. I'd also lean towards removing the comma after _Since_.